quic/s2n-quic-core/src/random.rs

```diff
@@ -75,6 +75,8 @@ mod tests {
 
     #[test]
     #[cfg_attr(miri, ignore)] // This test is too expensive for miri to complete in a reasonable amount of time
+    #[cfg_attr(kani, kani::proof)] 
+    #[cfg_attr(kani, kani::unwind(10))]
```

Comment:
This can be combined

```suggestion
    #[cfg_attr(kani, kani::proof, kani::unwind(10))]
```